reconcile/utils/oc.py

```diff
@@ -218,7 +218,11 @@ def project_exists(self, name):
 
     def new_project(self, namespace):
         cmd = ['new-project', namespace]
-        self._run(cmd)
+        try:
+            self._run(cmd)
+        except StatusCodeError as e:
+            if 'AlreadyExists' not in str(e):
```

Comment:
We have this type of check in the _run itself... can we move this there and pass on a custom exception here?

Reply:
Now I see the `if 'NotFound' in str(e):` below... yeah, we are mixing already.

Reply:
in `_run` there is the check only for `oc apply`. this pattern is common (expand lines above and below)